apps/website/src/content/docs/progressindicator.mdx

```diff
@@ -1,29 +1,90 @@
 ---
 title: Progress indicator
-description: Progress indicators express an unspecified wait time or display the length of a process.
+description: Progress indicators express the length of in-progress operations.
 thumbnail: ProgressIndicator
 ---
 
 <p>{frontmatter.description}</p>
 
-## Progress linear
+## Usage
 
-<Placeholder componentPageName='progresslinear--determinate' />
+There are two types of progress indicators which are linear and radial. Linear indicators show progress on a linear bar, while radial indicators display progress in a circular form.
+
+The default progress indicator is indeterminate. If a [value](#value) is passed, the indicator becomes determinate.
 
 <LiveExample src='ProgressLinear.main.jsx' truncate={false}>
   <AllExamples.ProgressLinearMainExample client:load />
 </LiveExample>
 
-### Props
+<LiveExample src='ProgressRadial.main.jsx' truncate={false}>
+  <AllExamples.ProgressRadialMainExample client:load />
+</LiveExample>
 
-<PropsTable path='@itwin/itwinui-react/esm/core/ProgressIndicators/ProgressLinear.d.ts' />
+### Value
 
-## Progress radial
+To specify a value for indeterminate indicator, pass a number into the `value` prop. This prop is available for both linear and radial indicators. If the `indeterminate` prop is set to `true`, this value will be ignored. This value must be inclusively between 0 and 100.
```

Comment:
Is "indeterminate" a typo?

Also some slight rewording/moving things around for readability and clarification around the purpose of `value`

```suggestion
To specify a value for determinate progress indicator, the `value` prop can be used. This prop is available for both linear and radial indicators. The value must be inclusively between `0` and `100`, representing what percentage of the task has completed.

If the `indeterminate` prop is set to `true`, this value will be ignored.
```

Reply:
I have made the change.

Reply:
Similar to https://github.com/iTwin/iTwinUI/pull/2271#discussion_r1784868852, this note about `indeterminate` could be prefixed with "Note:" and moved below the demo? Since there are multiple demos in this section, we can put the note below everything, at the bottom of this section.

Reply:
I have put this below the demo.